web/themes/custom/fmcg_material/templates/block/block--system-branding-block.html.twig

```diff
@@ -23,7 +23,7 @@
   {% endif %}
   {% if site_name %}
     <div class="navbar-title">
-      <a href="{{ path('<front>') }}" title="{{ 'Home'|t }}" rel="home">Christian Jobs</a>
+      <a href="{{ path('<front>') }}" title="{{ 'Home'|t }}" rel="home"></a>
```

Comment:
We can add FMCG Jobs here and hide it with css. It's better for blind people.

Reply:
```scss
.navbar-title a {
  text-indent: -1000px;
}
```